pom.xml

```diff
@@ -2654,6 +2654,16 @@
 
     <profile>
       <id>kubernetes</id>
+      <modules>
+        <module>resource-managers/kubernetes/core</module>
+      </modules>
+    </profile>
+
+    <profile>
+      <id>kubernetes-tests</id>
+      <activation>
+      <activeByDefault>false</activeByDefault>
+      </activation>
```

Comment:
Hi, @skonto . Could you use more specific PR title like 'Create `kubernetes-tests` profile and use the detected SCALA_VERSION`?

Reply:
BTW, there is a prior PR from @srowen . In this case, we prefer the older one.
- https://github.com/apache/spark/pull/22829

Reply:
@dongjoon-hyun we had a discussion on the list, I thought I would have a look at it. 
Anyway I can remove that part is it ok?

Reply:
No just keep this PR

Reply:
@srowen @dongjoon-hyun I will use `kubernetes-integration-tests` as the profile from the other pr.